cpp/src/parquet/metadata.cc

```diff
@@ -662,7 +665,7 @@ class FileMetaData::FileMetaDataImpl {
         encrypted_buffer->mutable_data());
     // Delete AES encryptor object. It was created only to verify the footer signature.
     aes_encryptor->WipeOut();
-    delete aes_encryptor;
+    aes_encryptor = nullptr;
```

Comment:
What's the point of this? `aes_encryptor` is just a local variable.

Reply:
This is to keep same behavior as before, but actually it's not need